readme.md

```diff
@@ -144,6 +144,62 @@ const emitter = require('./some-event-emitter');
 })();
 ```
 
+### pEvent.multiple(emitter, event, options)
+
+Wait for multiple event emissions. Returns an array.
+
+**Note**: The returned array will be mutated when the event is emitted.
```

Comment:
This is in the wrong place. It only applies to the `resolveImmediately` option.

Reply:
Right. I'll change that.